src/hooks/useSnackMessage.ts

```diff
@@ -95,7 +96,7 @@ function makeSnackbar(
             header,
             enqueueSnackbar,
             level,
-            persistent
+            snackInputs.persistent ? snackInputs.persistent : false
```

Comment:
If you want to eliminate nullish values just do `snackInputs.persistent ?? false` but it's not really necessary as for notistack default value is false already

Reply:
i reused the OptionsObject type here